src/cascadia/TerminalSettingsModel/TerminalSettings.idl

```diff
@@ -5,6 +5,12 @@ import "CascadiaSettings.idl";
 
 namespace Microsoft.Terminal.Settings.Model
 {
+    runtimeclass TerminalSettingsStruct

```

Comment:
Consider that this is part of TerminalSettingsModel's "public API" now, and that the name "TerminalSettingsStruct" doesn't _mean anything_ to a consumer of the library. To a C++ person, they're going to say "yeah? it's a struct? like _everything else in my language_". 

I'd suggest a rename... perhaps to `TerminalSettingsCreateResult`. Consider what it is _for_, not what it is _made of_ :smile: